todo-mvc/src/main.ts

```diff
@@ -26,23 +26,25 @@ app.loadDefinition({
 				keypress: todoInput
 			}
 		},
-		{
+		<ContainerListWidgetDefinition> {
```

Comment:
TypeScript isn't smart enough to match each individual definition object to one of the [`WidgetDefinition` types](https://github.com/dojo/app/pull/38/files#diff-3e8e21b91471e48a7df3e3deb6791992R311). If you cast the object it does perform type checking on whether the factory is a container list or container map.